api/tests/integration/infrastructure/repositories/sessions/session-repository_test.js

```diff
@@ -241,6 +241,51 @@ describe('Integration | Repository | Session', function () {
       expect(actualSession.certificationCandidates).to.be.empty;
     });
 
+    it('should return candidates complementary certifications', async function () {
+      // given
+      const session = databaseBuilder.factory.buildSession();
+      const pixPlusFoot = databaseBuilder.factory.buildComplementaryCertification({ name: 'Pix+Foot' });
+      const pixPlusRugby = databaseBuilder.factory.buildComplementaryCertification({ name: 'Pix+Rugby' });
+      const pixPlusTennis = databaseBuilder.factory.buildComplementaryCertification({ name: 'Pix+Tennis' });
+      const firstCandidate = databaseBuilder.factory.buildCertificationCandidate({
+        lastName: 'Jackson',
+        firstName: 'Michael',
+        sessionId: session.id,
+      });
+      const secondCandidate = databaseBuilder.factory.buildCertificationCandidate({
+        lastName: 'Stardust',
+        firstName: 'Ziggy',
+        sessionId: session.id,
+      });
+      databaseBuilder.factory.buildComplementaryCertificationSubscription({
+        certificationCandidateId: firstCandidate.id,
+        complementaryCertificationId: pixPlusRugby.id,
+      });
+      databaseBuilder.factory.buildComplementaryCertificationSubscription({
+        certificationCandidateId: secondCandidate.id,
+        complementaryCertificationId: pixPlusFoot.id,
+      });
+      databaseBuilder.factory.buildComplementaryCertificationSubscription({
+        certificationCandidateId: secondCandidate.id,
+        complementaryCertificationId: pixPlusTennis.id,
+      });
+      await databaseBuilder.commit();
+
+      // when
+      const actualSession = await sessionRepository.getWithCertificationCandidates(session.id);
+
+      // then
+      expect(
+        _.find(actualSession.certificationCandidates, { id: firstCandidate.id }).complementaryCertifications
+      ).to.deep.equal([domainBuilder.buildComplementaryCertification(pixPlusRugby)]);
+      expect(
+        _.find(actualSession.certificationCandidates, { id: secondCandidate.id }).complementaryCertifications
+      ).to.deep.equal([
+        domainBuilder.buildComplementaryCertification(pixPlusFoot),
+        domainBuilder.buildComplementaryCertification(pixPlusTennis),
+      ]);
+    });
```

Comment:
on trie par lastName, firstName, on peut donc faire:

```suggestion
const [firstCandidateFromSession, secondCandidateFromSession] = actualSession.certificationCandidates;
expect(firstCandidateFromSession.complementaryCertifications).to.deep.equal([domainBuilder.buildComplementaryCertification(pixPlusRugby)]);
expect(secondCandidateFromSession.complementaryCertifications.complementaryCertifications      ).to.deep.equal([
        domainBuilder.buildComplementaryCertification(pixPlusFoot),
        domainBuilder.buildComplementaryCertification(pixPlusTennis),
      ]);
```

Reply:
En effet, c'est modifié.